app/components/review_component/review_component.scss

```diff
@@ -64,6 +72,27 @@
   @include govuk-media-query($until: desktop) {
     margin-left: govuk-spacing(4);
   }
+
+  &--no-bullet {
+    list-style-type: none;
+    padding-left: 0;
+  }
+}
+
+.app-page-navigation__item {
```

Comment:
I know there are things in this file already, but generally a view components stylesheet should have everything scoped within the default class